commands/shooter/aimsystem.py

```diff
@@ -0,0 +1,106 @@
+from commands2 import CommandBase
+from wpilib import SmartDashboard
+from wpimath.geometry import Pose2d, Transform2d, Rotation2d
+from operatorinterface import Control2D
+
+from subsystems.shootingsubsystem import ShootingSubsystem
+import constants
+from util.angleoptimize import optimizeAngle
+from util.convenientmath import rotationFromTranslation
+
+
+class AimSystem(CommandBase):
+    def __init__(self, shoot: ShootingSubsystem, shooterOffset: Control2D) -> None:
+        SmartDashboard.putBoolean(constants.kShootingManualModeKey, False)
+        CommandBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.shoot = shoot
+        self.offset = shooterOffset
+        self.addRequirements([self.shoot])
+
+    def execute(self) -> None:
+        if SmartDashboard.getBoolean(constants.kShootingManualModeKey, False):
+            self.manualMode()
+        else:
+            self.normalMode()
```

Comment:
We can simplify things by making this command just perform automated aiming (normal mode) and having the `SpinMotor` (renamed to `AimShooterManually`) command handle the manual mode. If we want to enable/disable subsets of turret/hood/flywheel tracking, lets do it from SD entries `kAimTurretToTargetEnableKey`, `kAimHoodToTargetEnableKey`, `kAimFlywheelToTargetEnableKey`